selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -255,8 +255,6 @@ public void waitProjectExplorer(int timeout) {
       if (seleniumWebDriverHelper.isVisible(By.id("ide-loader-progress-bar"))) {
         fail("Known issue https://github.com/eclipse/che/issues/8468", ex);
       }
-
```

Comment:
In this case it's better to re-throw  exception ex for the sake of not hiding timeout exception.